<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>723</width>
    <height>438</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>860</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WebCam com Qt e Opencv</string>
  </property>
  <widget class="QLabel" name="lbFrames">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCam">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>START CAM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoad">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LOAD VIDEO</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinerX">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkEixoX">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Linha Pos Eixo X</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinerY">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkEixoY">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Linha Pos Eixo X</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
